docs/visual-basic/getting-started/breaking-changes-in-visual-studio.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Visual Basic Breaking Changes in Visual Studio"
+title: "Breaking Changes in Visual Studio"
```

Comment:
I think the original is better here, but then again, I think this is the article that we think should be retired.

Reply:
@gewarren, Reverted the change as this content may get retired soon.